meta-lmp-bsp/recipes-support/mfgtool-files/mfgtool-files_%.bbappend

```diff
@@ -34,6 +40,12 @@ do_deploy_prepend_mx6ull() {
     install -m 0644 ${DEPLOY_DIR_IMAGE}/u-boot.itb ${DEPLOYDIR}/${PN}/u-boot-mfgtool.itb
 }
 
+do_deploy_prepend_imx6ullevk() {
+    install -d ${DEPLOYDIR}/${PN}
+    install -m 0644 ${WORKDIR}/fuse.uuu ${DEPLOYDIR}/${PN}/fuse.uuu
+    install -m 0644 ${WORKDIR}/close.uuu ${DEPLOYDIR}/${PN}/close.uuu
```

Comment:
I also thought it would be good to have both bootloader.uuu and full_image.uuu refer to SPL-*.signed.
What do you think:

> do_compile_append_apalis-imx6-sec() {
>     sed -i 's/SPL.*/&.signed/g' bootloader.uuu
>     sed -i 's/SPL.*/&.signed/g' full_image.uuu
> }

Reply:
I think it might be a great idea. @ricardosalveti could you please, share your thoughts as well?

Reply:
Yes, it's safe to assume the files that will be flashed will all be appended with .signed.

Reply:
And to avoid duplicating the files, we can indeed manipulate it to have the correct file name.